kusama-guide/siteConfig.js

```diff
@@ -4,23 +4,23 @@ const isBuilding = process.env.BUILDING;
 const isPub = process.env.PUBLISHING;
 
 const siteConfig = {
-  title: 'Kusama Guide', // Title for your website.
+  title: 'Guide', // Title for your website.
   tagline: 'One-stop-shop for Kusama information.',
-  url: isPub ? "https://guide.kusama.network" : "", // Your website URL
-  baseUrl: isBuilding ? baseUrlPattern : "/",
-  projectName: isPub ? "kusama-guide-hosting" : "",
-  organizationName: 'w3f',
-  cname: "guide.kusama.network",
+  url: isPub ? "https://lsaether.github.io/" : "", // Your website URL
+  baseUrl: isBuilding ? baseUrlPattern : (isPub ? "/test/" : "/"),
+  projectName: isPub ? "test" : "",
+  organizationName: 'lsaether',
```

Comment:
```suggestion
   url: isPub ? "https://guide.kusama.network" : "", // Your website URL
  baseUrl: isBuilding ? baseUrlPattern : "/",
  projectName: isPub ? "kusama-guide-hosting" : "",
  organizationName: 'w3f',
  cname: "guide.kusama.network",
```

This was my testing configuration, replaced now with the live settings.